packages/@atjson/document/src/serialize.ts

```diff
@@ -669,7 +669,11 @@ export function serialize(
   };
 }
 
```

Comment:
What typing bugs were popping up here?

Reply:
you couldn't pass typed blobs into deserialize because all of the internal block/mark types were interpreted as specifically `Block<null>` or `Mark<null>`